openpype/hosts/fusion/plugins/create/create_workfile.py

```diff
@@ -0,0 +1,129 @@
+            return
+
+        instance = CreatedInstance(
+            family=self.family,
+            subset_name=data["subset"],
+            data=data,
+            creator=self
+        )
+        instance.transient_data["comp"] = comp
+        instance.transient_data["tool"] = None
+
+        self._add_instance_to_context(instance)
+
+    def update_instances(self, update_list):
+        for update in update_list:
+            instance = update.instance
+            comp = instance.transient_data["comp"]
+            if not hasattr(comp, "SetData"):
+                # Comp is not alive anymore, likely closed by the user
+                self.log.error("Workfile comp not found for existing instance."
+                               " Comp might have been closed in the meantime.")
+                continue
+
+            # TODO: It appears sometimes this could be 'nested'
+            # Get the new values after the changes by key, ignore old value
+            new_data = {
+                key: new for key, (_old, new) in update.changes.items()
+            }
+            self._imprint(comp, new_data)
+
+    def create(self, options=None):
+
+        comp = api.get_current_comp()
+        if not comp:
+            self.log.error("Unable to find current comp")
+            return
+
+        # TODO: Is this really necessary?
+        # Force kill any existing "workfile" instances
+        for instance in self.create_context.instances:
+            if instance.family == self.family:
+                self.log.debug(f"Removing instance: {instance}")
+                self._remove_instance_from_context(instance)
+
+        project_name = legacy_io.Session["AVALON_PROJECT"]
+        asset_name = legacy_io.Session["AVALON_ASSET"]
+        task_name = legacy_io.Session["AVALON_TASK"]
+        host_name = legacy_io.Session["AVALON_APP"]
+
+        asset_doc = get_asset_by_name(project_name, asset_name)
+        subset_name = self.get_subset_name(
+            self.default_variant, task_name, asset_doc,
+            project_name, host_name
+        )
+        data = {
+            "asset": asset_name,
+            "task": task_name,
+            "variant": self.default_variant
+        }
+        data.update(self.get_dynamic_data(
+            self.default_variant, task_name, asset_doc,
+            project_name, host_name
+        ))
+
+        instance = CreatedInstance(
+            self.family, subset_name, data, self
+        )
+        instance.transient_data["comp"] = comp
+        instance.transient_data["tool"] = None
+        self._add_instance_to_context(instance)
+
+        self._imprint(comp, data)
+
+    def get_icon(self):
+        return qtawesome.icon("fa.file-o", color="white")
+
+    def _imprint(self, comp, data):
+
+        # TODO: Should this keys persist or not? I'd prefer not
+        # Do not persist the current context for the Workfile
+        for key in ["variant", "subset", "asset", "task"]:
+            data.pop(key, None)
```

Comment:
The After Effects workfile creator "overrides" asset, task and subset with the current context if it differs from what's stored in the scene. If that's the case - why would we store it to begin with?